aten/src/ATen/native/cuda/ReduceSumProdKernel.cu

```diff
@@ -21,6 +21,14 @@ void prod_kernel_impl(TensorIterator& iter) {
   }), 1);
 }
 
+template <typename scalar_t, typename acc_t=scalar_t, typename out_t=scalar_t>
+void nanprod_kernel_impl(TensorIterator& iter) {
+  gpu_reduce_kernel<scalar_t, out_t>(iter, func_wrapper<out_t> ([]GPU_LAMBDA(acc_t a, acc_t b) -> acc_t {
+    std::cout << a << ", " << b << std::endl;
```

Comment:
debug code

Reply:
Sorry, thank you for pointing this out. Removed.